docs/API.md

```diff
@@ -481,6 +481,29 @@ formatted as IPv4-mapped IPv6 addresses). By default (-1) this option is not set
 and the platform default value is used.
 ---
 
+| OptName               | Since | Binding | Type      | Units  | Default  | Range  |
+| --------------------- | ----- | ------- | --------- | ------ | -------- | ------ |
+| `SRTO_KMREFRESHRATE`  | 1.3.2 | pre     | `int32_t` | pkts   | 0x1000000| 0..unlimited |
+
+- **[GET or SET]** - number of packets to be transmitted after which the encryption
+key will be switched to the new one. The process involves "preannounce" when the new
+key is sent to the receiver, and "decommission" when the old key is no longer used;
```

Comment:
key is sent to the receiver; this is followed by a "decommissioning" when the old key is no longer used;